spi/common/core-spi/src/main/java/org/eclipse/edc/spi/types/domain/transfer/DataFlowRequest.java

```diff
@@ -158,6 +165,11 @@ public Builder destinationDataAddress(DataAddress destination) {
             return this;
         }
 
+        public Builder transferType(String transferType) {
+            request.transferType = transferType;
```

Comment:
I presume this is a required property, and if so there should be a validation in the `build()` method, plus a test asserting this

Reply:
It's not a mandatory property for backward compatibility